pkg/cosign/tsalog.go

```diff
@@ -0,0 +1,75 @@
+package cosign
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"github.com/sigstore/cosign/v2/internal/pkg/cosign/tsa"
+	"os"
+
+	"github.com/sigstore/cosign/v2/pkg/cosign/env"
+	"github.com/sigstore/sigstore/pkg/tuf"
+)
+
+var tsaLeafCertStr = `tsa_leaf.crt.pem`
+var tsaRootCertStr = `tsa_root.crt.pem`
+var tsaIntermediateCertStrPattern = `tsa_intermediate_%d.crt.pem`
+
+// GetTSACerts retrieves trusted TSA certificates from the embedded or cached
+// TUF root. If expired, makes a network call to retrieve the updated targets.
+// By default, the certificates come from TUF, but you can override this for test
+// purposes by using an env variable `SIGSTORE_TSA_CERTIFICATE_FILE`. If using
+// an alternate, the file should be in PEM format.
+func GetTSACerts(ctx context.Context) (leaves [][]byte, intermediates [][]byte, roots [][]byte, err error) {
```

Comment:
Should we return `x509.Certificate` structs instead of byte arrays? This will cut down on repetition in the verify files if you handle unmarshalling from PEM in this function.

Reply:
that's a good point, i'll look into that. thanks for the feedback!